common/djangoapps/third_party_auth/settings.py

```diff
@@ -85,3 +84,5 @@ def apply_settings(django_settings):
         'social.apps.django_app.context_processors.backends',
         'social.apps.django_app.context_processors.login_redirect',
     )
+
+    django_settings.OPTIONAL_USER_FIELDS = ['first_name', 'last_name', 'fullname']
```

Comment:
What app uses this setting? I can't tell. If this is affecting the default required fields for all new users on the platform, we should find another approach. We can't be marking these fields as optional in general.